bucket/retdec.json

```diff
@@ -0,0 +1,60 @@
+{
+    "version": "4.0",
+    "description": "A retargetable machine-code decompiler based on LLVM.",
+    "homepage": "https://github.com/avast/retdec",
+    "license": "MIT",
+    "notes": [
+        "The python executables were replaced with *.cmd by the shim creation.",
+        "You could install the following programs:",
+        " - UPX (Optional: if you want to use UPX unpacker in the preprocessing stage);",
+        " - Graphviz (Optional: if you want to generate call or control flow graphs)."
+    ],
+    "suggest": {
+        "Python 3.4+": "python",
+        "Visual C++ Redistributable": "extras/vcredist2022"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/avast/retdec/releases/download/v4.0/retdec-v4.0-windows-64b.7z",
+            "hash": "5fc667da3af3530cf3d393f96627987c3e46da62f4b426e78cdcc25a79416a31"
+        },
+        "32bit": {
+            "url": "https://github.com/avast/retdec/releases/download/v4.0/retdec-v4.0-windows-32b.7z",
+            "hash": "c9b6fc91007c2158209ac730d981a99006a960a27e3b0f54e5618872f34e1afe"
+        }
+    },
+    "extract_dir": "retdec",
+    "bin": [
+        "bin\\retdec-archive-decompiler.py",
+        "bin\\retdec-ar-extractor.exe",
+        "bin\\retdec-bin2llvmir.exe",
+        "bin\\retdec-bin2pat.exe",
+        "bin\\retdec-config.exe",
+        "bin\\retdec-config.py",
+        "bin\\retdec-decompiler.py",
+        "bin\\retdec-fileinfo.exe",
+        "bin\\retdec-fileinfo.py",
+        "bin\\retdec-getsig.exe",
+        "bin\\retdec-idb2pat.exe",
+        "bin\\retdec-llvmir2hll.exe",
+        "bin\\retdec-macho-extractor.exe",
+        "bin\\retdec-pat2yara.exe",
+        "bin\\retdec-signature-from-library-creator.py",
+        "bin\\retdec-stacofin.exe",
+        "bin\\retdec-unpacker.exe",
+        "bin\\retdec-unpacker.py",

```

Comment:
```suggestion
```

exe for this already exists

Reply:
The *.py files are wrapper for the *.exe

Reply:
Then the exe file should be removed from the array

Reply:
But it could also be useful, I don't know why there are two choice

Reply:
Only one of them can be used, because of the preference order in the PATHEXT variable - so there's no use of two shims. The other will never get invoked.

Reply:
I think this applies to all these duplicates

Reply:
Hmm, okay. What about the help texts of retdec-unpacker?

Reply:
They are more similar, but it seems that the exe has more functions
[out-unpacker.py-help.txt](https://github.com/ScoopInstaller/Main/files/9567021/out-unpacker.py-help.txt)
[out-unpacker.exe-help.txt](https://github.com/ScoopInstaller/Main/files/9567022/out-unpacker.exe-help.txt)

Reply:
Okay